config/manifests/bases/oadp-operator.clusterserviceversion.yaml

```diff
@@ -314,6 +314,164 @@ spec:
+        path: completionTimestamp
+      - description: Message is a message about the current status of the BackupRepository.
+        displayName: Message
+        path: message
+      - description: Node is name of the node where the DataDownload is processed.
+        displayName: Node
+        path: node
+      - description: Progress holds the total number of bytes of the snapshot and
+          the current number of restored bytes. This can be used to display progress
+          information about the restore operation.
+        displayName: Progress
+        path: progress
+      - description: StartTimestamp records the time a restore was started. The server's
+          time is used for StartTimestamps
+        displayName: StartTimestamp
+        path: startTimestamp
+      version: v2alpha1
+    - description: DataUpload represents a data upload of a volume snapshot. There
+        is one DataUpload created per CSI snapshot, which is handled by the Data mover.
+      displayName: DataUpload
+      kind: DataUpload
+      name: datauploads.velero.io
+      statusDescriptors:
+      - description: CompletionTimestamp records the time a backup was completed.
+          Completion time is recorded even on failed backups. Completion time is recorded
+          before uploading the backup object. The server's time is used for CompletionTimestamps
+        displayName: CompletionTimestamp
+        path: completionTimestamp
+      - description: DataMoverResult stores data-mover-specific information as a result
+          of the DataUpload.
+        displayName: DataMoverResult
+        path: dataMoverResult
+      - description: Message is a message about the DataUpload's status.
+        displayName: Message
+        path: message
+      - description: Node is name of the node where the DataUpload is processed.
+        displayName: Node
+        path: node
+      - description: Path is the full path of the snapshot volume being backed up.
+        displayName: Path
+        path: path
+      - description: Phase is the current state of the DataUpload.
+        displayName: Phase
+        path: phase
+      - description: Progress holds the total number of bytes of the volume and the
+          current number of backed up bytes. This can be used to display progress
+          information about the backup operation.
+        displayName: Progress
+        path: progress
+      - description: SnapshotID is the identifier for the snapshot in the backup repository.
+        displayName: SnapshotID
+        path: snapshotID
+      - description: StartTimestamp records the time a backup was started. Separate
+          from CreationTimestamp, since that value changes on restores. The server's
+          time is used for StartTimestamps
+        displayName: StartTimestamp
+        path: startTimestamp
+      version: v2alpha1
+    - description: VolumeSnapshotBackup is the Schema for the volumesnapshotbackups
```

Comment:
VSB and VSR do not exist in master anymore, this can be applied to others branch